scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/Messages.scala

```diff
@@ -29,8 +29,6 @@ object Messages {
   val InvalidImplicit =
     "`implicit' modifier can be used only for values, variables, methods and classes"
   val InvalidImplicitTrait = "traits cannot be implicit"
-  val InvalidImplicitClass = "classes cannot be implicit"
   val InvalidAbstract = "`abstract' modifier can be used only for classes"
-  val InvalidOverrideClass = "`override' modifier not allowed for classes"
```

Comment:
Looks like this is not allowed for Scala 2.13.x, but maybe it's not really useful to add a new dialect option.

Reply:
we could check for significant indentation flag, or, like we frequently do, just err on the side of more permissive parsing, assuming the compiler is ok.